tests/console/validate_raid.pm

```diff
@@ -105,13 +114,20 @@ sub prepare_test_data {
             $num_raid_arrays = @raid_arrays;
             @raid            = (($raid_level, $raid0, $raid1), @raid_detail);
         }
-    }
-    else {
-        @partitioning = (
-            $raid_partitions_2_arrays, $hard_disks, $linux_raid_member_2_arrays,
-            $btrfs, $swap,
-        );
-        @raid = (($raid_level, $raid0), @raid_detail);
+        else {
+            @partitioning = (
+                $raid_partitions_2_arrays, $hard_disks, $linux_raid_member_2_arrays,
+                $btrfs, $swap,
+            );
+            if (is_sle('<=15-SP1') || get_var('STAGING') ||
+                is_leap() || (is_tumbleweed() && check_var('FLAVOR', 'NET'))) {
```

Comment:
I am wondering why NET flavor should be handled separately?

Reply:
DVD TW have both arrays in the same raid while NET TW has this 'special' RAID0 for swap.

Reply:
Ok, I see it in openQA. But it looks like a bug to me. DVD and NET should have the same behavior, no?

Reply:
Now I have check better, I can see that this is an artificial difference that we have created, we use different module to do the raid, an OOP module `installation/partitioning/raid_gpt` and a legacy one `installation/partitioning_raid`. I would leave that out-of-scope of this PR and I will share with the team to fix in the future.